pkg/query/logical/plan_indexscan.go

```diff
@@ -107,12 +108,13 @@ type indexScan struct {
 }
 
 func (i *indexScan) Execute(ec executor2.ExecutionContext) ([]data.Entity, error) {
-	var chunkSet common.ChunkIDs
+	var chunkSet posting.List
 	for _, exprs := range i.conditionMap {
 		// TODO: Discuss which metadata should be used!
```

Comment:
@lujiajing1126 IndexRule metadata should be selected here

Reply:
But as I understand, a IndexRule probably can be bounded to several Series. So there is a 1:N relationship between IndexRule and Series

Reply:
Sorry, it should be the `RuleBinding` which represents the relationship between `IndexRule` and `Series`